MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include README.md
+recursive-include cmsplugin_feed_ai *.*
```

Comment:
This is way too broad. Just make it `*.html` instead of `*.*`, to include templates.

You can then remove the other lines (including the one that packages `README.md` in.)